### PR TITLE
Feature/pai fe case study

### PR DESCRIPTION
**Pull request by archanaganesan11**
Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-fe-case-study--pricefx-eds--pricefx.hlx.live/style-guide/components/case-study

### CODE REVIEW

File: blocks/case-study/case-study.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.case-study .grid-container-3 , .case-study .grid-container-2 {
+  display: flex;
+  flex-direction: column; 
+  overflow: hidden;
+  justify-content: space-between; 
+}
+
+.case-study .grid-container-3 .image-1 , .grid-container-3 .image-2 , .grid-container-3 .image-3 , .grid-container-2 .image-1 , .grid-container-2 .image-2 {
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:12]: Please move each selector into a new line. Also prefix class is missing.

File: blocks/case-study/case-study.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
+.case-study {
+    display: flex;
+}
+
+.case-study .grid-container-3 , .case-study .grid-container-2 {
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:5]: Please move each selector into a new line

File: blocks/case-study/case-study.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  flex-direction: column; 
+  overflow: hidden;
+  justify-content: space-between; 
+}
+
+.case-study .grid-container-3 .image-1 , .grid-container-3 .image-2 , .grid-container-3 .image-3 , .grid-container-2 .image-1 , .grid-container-2 .image-2 {
+  position: relative;
+}
+
+.case-study .grid-container-3 img , .grid-container-2 img {
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:16]: Please move each selector into a new line

File: blocks/case-study/case-study.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  const [image1, title1, text1, image2, , title2, text2, image3, title3, text3, swapRight] = block.children;
+
+  const container = document.createElement('div');
+
+  const firstImageContainer = createImageContainer(image1, title1, text1, 'image-1');
+
+  const secondImageContainer = createImageContainer(image2, title2, text2, 'image-2');
+
+  if (
+    title1.textContent.replace(/\s+/g, '') &&
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.js:40]: change `replace(/\s+/g, ''` to `trim()` in all instances

File: blocks/case-study/case-study.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+.case-study .grid-container-3 ,
+.case-study .grid-container-2 {
+  display: flex;
+  flex-direction: column; 
+  overflow: hidden;
+  justify-content: space-between; 
+}
+
+.case-study .grid-container-3 .image-1 ,
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:13]: Can you please remove the spacing between `image-1` and `,` in all instances?

File: blocks/case-study/case-study.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+.case-study .grid-container-2 .image-2 {
+  position: relative;
+}
+
+.case-study .grid-container-3 img ,
+.case-study .grid-container-2 img {
+  height: auto;
+}
+
+.case-study .title-text  {
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:26]: Remove a single space here

File: blocks/case-study/case-study.css [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  text-align: left;
+  opacity: 0.7518;
+  background-color: var(--c-gray-med-2);
+  color: var(--c-white);
+  padding-left: 20px;
+  box-sizing: border-box;
+  margin: 20px;
+}
+
+@media (width<= 985px){
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:62]: Add a space b/w `)` and `{`

File: blocks/case-study/case-study.css [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  .case-study .half-container {
+    flex-direction: column;
+     height: 767px;
+  }
+
+  .case-study .half-container img {
+    height: 381px;
+  }
+
+  .case-study .grid-container-3 .image-1 img{
```
Comment (Vaitheesaran-nagalingam) [inline blocks/case-study/case-study.css:86]: Add a space `img` and `{`

**Review by Vaitheesaran-nagalingam — COMMENTED** [review]
Please check the CSS file formatting.

**Review by muneeskumar-rajendran — APPROVED** [review]

**Pull request merged**